src/Test/L0/Listener/Configuration/Testable/TestWindowsServiceControlManager.cs

```diff
@@ -0,0 +1,28 @@
+using System.ServiceProcess;
+
+using Microsoft.VisualStudio.Services.Agent.Listener.Configuration;
+
+namespace Test.L0.Listener.Configuration.Testable
+{
+    public class TestWindowsServiceControlManager : WindowsServiceControlManager
+    {
+        public ServiceController SelectedServiceController { get; set; }
+
+        protected override void InstallService(
```

Comment:
I see, thanks! I'm not interested in the name, whatever name you think is appropriate is fine. Basically I just don't want diverging patterns for solving unit-testability.